src/commands/deploy/showDeployCompletedMessage.ts

```diff
@@ -9,25 +9,51 @@ import { AppServiceDialogResponses } from "../../constants";
 import { SiteTreeItem } from "../../explorer/SiteTreeItem";
 import { ext } from "../../extensionVariables";
 import { localize } from "../../localize";
+import * as workspaceUtil from '../../utils/workspace';
+import { uploadAppSettings } from "../appSettings/uploadAppSettings";
 import { startStreamingLogs } from '../logstream/startStreamingLogs';
 
-export function showDeployCompletedMessage(node: SiteTreeItem): void {
+export async function showDeployCompletedMessage(node: SiteTreeItem): Promise<void> {
     const message: string = localize('deployCompleted', 'Deployment to "{0}" completed.', node.client.fullName);
     ext.outputChannel.appendLog(message);
     const browseWebsiteBtn: MessageItem = { title: localize('browseWebsite', 'Browse Website') };
     const streamLogs: MessageItem = { title: localize('streamLogs', 'Stream Logs') };
+    const uploadSettingsBtn: MessageItem = { title: localize('uploadMessage', 'Upload Local Settings') };
 
-    // don't wait
-    void window.showInformationMessage(message, browseWebsiteBtn, streamLogs, AppServiceDialogResponses.viewOutput).then(async (result: MessageItem | undefined) => {
-        await callWithTelemetryAndErrorHandling('postDeploy', async (context: IActionContext) => {
-            context.telemetry.properties.dialogResult = result?.title;
-            if (result === AppServiceDialogResponses.viewOutput) {
-                ext.outputChannel.show();
-            } else if (result === browseWebsiteBtn) {
-                await node.browse();
-            } else if (result === streamLogs) {
-                await startStreamingLogs(context, node);
-            }
+    const filesByFileExt = await workspaceUtil.findFilesByFileExtension(undefined, "env");
```

Comment:
Yeah, we don't have logic to find the .env file, we just assume the `envFileName` to be `.env`.  We have logic in `uploadAppSettings.ts` to let users select which .env file they want, but by default it just shows the `{workspaceFolder}/.env`.

Based off of this https://gist.github.com/ericelliott/4152984 we may be able to consider .env and extension since they're using production.env, dev.env, staging.env as examples.